x-pack/plugins/actions/server/usage/actions_telemetry.ts

```diff
@@ -31,7 +31,12 @@ export async function getTotalCount(
       init_script: 'state.types = [:]',
       map_script: `
         String actionType = doc['action.actionTypeId'].value;
-        state.types.put(actionType, state.types.containsKey(actionType) ? state.types.get(actionType) + 1 : 1);
+        if (actionType =~ /.gen-ai/) {
```

Comment:
Did you intend to match "(any character)get-ai"?  Otherwise, I think you need a `\` in front of the `.`, assuming painless regexp's work the way you would expect.